### PR TITLE
Minor test improvements

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes:

* Disable stats push service when testing as this can lead to intermittent timing failures
* Fix case where tests run twice if failing
* Disable overriding the `openid_connect_providers` config in `:test`
* Add ExCoveralls to properly test `fz_common`
* Increase `fz_common` test coverage

### CODE REVIEW

File: config/runtime.exs [hunk 1]
```diff
@@ -255,7 +255,7 @@ end
 # OIDC Auth
 auth_oidc_env = System.get_env("AUTH_OIDC")
 
-if auth_oidc_env do
+if config_env() != :test && auth_oidc_env do
```
Comment (princemaple) [inline config/runtime.exs:258]: Does test env have this environment variable `AUTH_OIDC`?

**Comment by princemaple** [comment]
The failure is caused by this, I think 😅

Coverage test failed, threshold not met:

    Coverage:   42.62%
    Threshold:  90.00%

File: config/runtime.exs (hunk 1 above)
Comment (jamilbk) [reply config/runtime.exs:258]: Yeah, `.env` is env-agnostic (at least for me). Maybe we should clear these env vars for the `test` env, or update `runtime.exs` to be skip env vars when `config_env() == :test`? Would prevent other env vars from causing bugs in test.

**Comment by jamilbk** [comment]
> The failure is caused by this, I think 😅
> 
> Coverage test failed, threshold not met:
> 
> ```
> Coverage:   42.62%
> Threshold:  90.00%
> ```

Welp... 😶 strange. Maybe the threshold settings have changed? I get this when running `mix coveralls.html --umbrella` locally:

```
[TOTAL]  75.3%
```

File: config/runtime.exs (hunk 1 above)
Comment (princemaple) [reply config/runtime.exs:258]: Oh OK I don't use .env that's why I haven't discovered this issue. I understand now - this is for local test.

**Review by princemaple — DISMISSED** [review]
Changes are 👍 We just need to fix the coverage thing.

**Comment by princemaple** [comment]
> Welp... 😶 strange.

It seems to check each app. The 42.62% was reported on FzCommon

**Comment by jamilbk** [comment]
> > Welp... 😶 strange.
> 
> 
> 
> It seems to check each app. The 42.62% was reported on FzCommon

👍 I'll work on this now.

**Comment by princemaple** [comment]
image

Much better.

**Pull request merged**